query/cloud_test.go

```diff
@@ -0,0 +1,39 @@
+//go:build cloud
+
+/*
+ * Copyright 2023 Dgraph Labs, Inc. and Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package query
+
+import (
+	"os"
+	"testing"
+
+	"github.com/dgraph-io/dgraph/dgraphtest"
+)
+
+func TestMain(m *testing.M) {
+	c, err := dgraphtest.NewDCloudCluster()
```

Comment:
Maybe add a comment that it needs `TEST_DGRAPH_CLOUD_CLUSTER_URL` and `TEST_DGRAPH_CLOUD_CLUSTER_TOKEN` in `env`. I understand they'd see the error, but keeping a comment will be helpful here as well.

Reply:
I will add a README when we setup the CI. I will create it in the next PR.